chartbeat.php

```diff
@@ -286,28 +296,34 @@ function add_chartbeat_head() {
 	echo "\n<script type=\"text/javascript\">var _sf_startpt=(new Date()).getTime()</script>\n";
 }
 
+function add_chartbeat_config(){
+		
+		$cb_configs = chartbeat_configs();?>
+<script type="text/javascript">
+			var _sf_async_config={};
+			_sf_async_config.uid = <?php echo intval( $user_id ); ?>;
+			_sf_async_config.domain = "<?php echo $cb_configs["domain"]; ?>";
+			_sf_async_config.useCanonical = <?php echo $cb_configs["use_canonical"]; ?>;
+<?php
+		$enable_newsbeat = get_option('chartbeat_enable_newsbeat');
+		if ($enable_newsbeat) { ?>
+			_sf_async_config.authors = "<?php echo $cb_configs["author"] ?>";
+			_sf_async_config.sections = "<?php echo $cb_configs["sections"]?>";
```

Comment:
They get esc_js when the array is created. https://github.com/chartbeat-labs/wordpress/blob/fbia_support/chartbeat.php#L260